VMEncryption/main/DiskUtil.py

```diff
@@ -887,10 +890,15 @@ def get_azure_data_disk_controller_and_lun_numbers(self, dev_items_real_paths):
         """
         Return the controller ids and lun numbers for data disks that show up in the dev_items
         """
+        list_devices = []
+        is_os_nvme, root_device_path_nvme = self.is_os_disk_nvme()
 
+        if is_os_nvme:
```

Comment:
Is it true to say that if OS disk is NVMe then other all disks must be NVMe.

Reply:
Yes. For the new ASAP SKU's both OS and Data disk will be NVMe. Resource and BEK disk are SCSI.